stories/busy-indicator/__snapshots__/busy-indicator.stories.storyshot

```diff
@@ -34,6 +34,16 @@ exports[`Storyshots Components/Busy Indicator Contrast Mode 1`] = `
 </div>
 `;
 
+exports[`Storyshots Components/Busy Indicator Extended Busy Indicator 1`] = `
```

Comment:
empty storyshots, the opening backquote \` should be next to the first element, say `<div>` in the stories, otherwise the storyshots don't generate at all.